src/styles/utils/vars.scss

```
@@ -0,0 +1,9 @@
+$calendar__day-size: 120px;
+$days:
+  "mon" 0,
+  "tue" #{$calendar__day-size + 1px},
+  "wed" #{$calendar__day-size * 2 + 2px},
+  "thu" #{$calendar__day-size * 3 + 3px},
+  "fri" #{$calendar__day-size * 4 + 4px},
+  "sat" #{$calendar__day-size * 5 + 5px},
+  "sun" #{$calendar__day-size * 6 + 6px};
```

let's use 'days' like this, so you will calculate the needed margin inside @each only once

```suggestion
$days:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```